HW03-VaughtK/hw03_VaughtK.py

```diff
@@ -0,0 +1,196 @@
+
+#################################################################################
+# Author: Kai Vaught
+# Username: VaughtK
+# Assignment: Homework 3
+# Purpose: To learn how branches work.
+# Google Doc Link: https://docs.google.com/document/d/1OvsmR456y61n3UzwBlIlVE13IBz0tF39Vyv10F24ZGI/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+#
+#
+#################################################################################
+
+import turtle
+
+t = turtle.Turtle()
+
+turtle2 = turtle.Turtle()
+
+def draw_house():
```

Comment:
You likely put the turtle above outside of main because of scoping issues in here. You just need to pass that turtle in:

```
def draw_house(t):
    ...

def main():
    t = turtle.Turtle()
    draw_house(t)
```

Reply:
Fix this for all of your functions.